src/content/indigenous-heritage-of-saint-martin.md

```diff
@@ -0,0 +1,27 @@
+---
+title: The Rich Indigenous Heritage of Saint-Martin, A Tapestry of Arawak and Carib Culture
+author: Rayan LOUISY
+type: history

```

Comment:
- [ ] the `type` field needs to be changed to `category` for every article since `type` is a reserved field for [contentlayer](https://www.contentlayer.dev/)
- [ ] Optional - you can add a `profile` field so your profile pic would pop up
heres an example of how everything would look:
```yaml
title: The Rich Indigenous Heritage of Saint-Martin, A Tapestry of Arawak and Carib Culture
author: Rayan LOUISY
# this file is under the public directory
profile: /rayan-louisy/profile.png
# field is now category instead of type
category: history
thumbnail: /sxm2.jpg
```